lisp/ess-s-lang.el

```diff
@@ -777,21 +774,21 @@ If 'sp-for-help-changed?' process variable is non-nil or
 `ess-help-topics-list' is nil, (re)-populate the latter and
 return it.  Otherwise, return `ess-help-topics-list'."
   (with-ess-process-buffer nil
-    (ess-write-to-dribble-buffer
-     (format "(ess-get-help-topics-list %s) .." name))
-    (ess-help-r--check-last-help-type)
-    (cond
-     ;; (Re)generate the list of topics
-     ((or (not ess-help-topics-list)
-          (ess-process-get 'sp-for-help-changed?))
-      (ess-process-put 'sp-for-help-changed? nil)
-      (setq ess-help-topics-list
-            (ess-uniq-list
-             (append (ess-get-object-list name 'exclude-1st)
-                     (ess-get-help-files-list)
-                     (ess-get-help-aliases-list)))))
-     (t
-      ess-help-topics-list))))
+                           (ess-write-to-dribble-buffer
```

Comment:
Why the whithespace change?

Reply:
Sorry, didn't notice that. Should be fixed now.